u19_pipeline/ephys.py

```diff
@@ -48,17 +48,35 @@ class SkullReference(dj.Lookup):
 
 
 @schema
-class EphysSession(dj.Manual):
+class EphysRecording(dj.Computed):
     definition = """
     # General information of an ephys session
-    -> acquisition.Session
+    -> recording.Recording
     ---
-    ephys_directory: varchar(255)      # the absolute directory where the ephys data for this session will be stored in bucket
     """
+    @property
+    def key_source(self):
+        return recording.Recording & {'recording_modality': 'ephys'}
+
+    def make(self, key):
+        self.insert1(key)
+
+
+@schema
+class EphysSorting(dj.Computed):
```

Comment:
```suggestion
class EphysProcessing(dj.Computed):
```

Perhaps we could call this table `EphysProcessing` since there is a table in `element-array-ephys` called `Clustering` which I believe is a synonym for `Sorting`?